client/settings/dpcs-settingspanel/panel_evoker.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ "$BASH_SOURCE" == "$0" ]]
+then 
+	sudo -s python2 settingspanel.py
```

Comment:
Plsease do not use `sudo`. Let the user call the script as `sudo ./panel_evoker.sh` if need be. But consider storing per-user configuration instead of running the script as root.